File: src/color.rs
pub struct ANSIColors;

impl ANSIColors {
    pub const GREEN: &'static str = "\u{001B}[32m";
    pub const BLUE: &'static str = "\u{001B}[34m";
    pub const YELLOW: &'static str = "\u{001B}[33m";
    pub const MAGENTA: &'static str = "\u{001B}[35m";
    pub const CYAN: &'static str = "\u{001B}[36m";
    pub const RESET: &'static str = "\u{001B}[0m";
}

pub fn colorize(text: &str, color: &str) -> String {
    format!("{}{}{}", color, text, ANSIColors::RESET)
}

File: src/color.go
package main

type ANSIColors struct{}

func (ANSIColors) green(text string) string {
    return "\033[32m" + text + "\033[0m"
}

func (ANSIColors) blue(text string) string {
    return "\033[34m" + text + "\033[0m"
}

func (ANSIColors) yellow(text string) string {
    return "\033[33m" + text + "\033[0m"
}

func (ANSIColors) magenta(text string) string {
    return "\033[35m" + text + "\033[0m"
}

func (ANSIColors) cyan(text string) string {
    return "\033[36m" + text + "\033[0m"
}

